scripts/split_codes.rb

```diff
@@ -29,7 +32,11 @@ def fix_image_paths(string)
     end
 
     def fix_github_issues(string)
-      swap_term_refs(fix_image_paths(string))
+      refs_patch(swap_term_refs(fix_image_paths(string)))
+    end
+
+    def refs_patch(string)
+      string.gsub(/IEC 60050-191:1990/, '<<IEV191_1990>>')
```

Comment:
Why not replace with the generic IEV anchor instead of an individual 60050 part?

Reply:
These are not term references, they are references to the entire IEV part document. A generic IEV anchor only works if provided a full term reference.

Reply:
OK